.gitignore

```diff
@@ -152,6 +152,10 @@ rc/trafficserver.service
 
 tsxs
 
+test_metrics
```

Comment:
Are these two really needed? It seems like we have lines for them.
https://github.com/webdriverrr/trafficserver/blob/9abdf95fb757ed8167fa075e600d890c184b316d/.gitignore#L56
https://github.com/webdriverrr/trafficserver/blob/9abdf95fb757ed8167fa075e600d890c184b316d/.gitignore#L110

Reply:
Yes indeed these two are extras and can be removed.